docs/src/data_documentation.md

```diff
@@ -311,6 +311,9 @@ This file contains cost and performance parameters for various generators and ot
 |VRE_STOR | {0, 1}, Flag to indicate membership in set of co-located variable renewable energy resources (onshore wind and utility-scale solar PV) and storage resources (either short- or long-duration energy storage with symmetric or asymmetric charging or discharging capabilities).|
 ||VRE_STOR = 0: Not part of set (default) |
 ||VRE_STOR = 1: Co-located VRE and storage (VRE-STOR) resources. |
+|BIOMASS | {0, 1}, Flag to indicate if generator use biomass as feedstock (optional input column).|
+||BIOMASS = 0: Not part of set (default) |
+||BIOMASS = 1: Use biomass as fuel.|
```

Comment:
Is BIOMASS a required column? If not, please move it out of the 'mandatory columns' table. 

* I'd suggest renaming it as 'Biomass' or 'UsesBiomass', since it's not a specific generator type, right?

Reply:
ok will change that to "Biomass", I've moved them to Table 6